examples/starrocks/deploy_a_starrocks_cluster_with_all_features.yaml

```diff
@@ -0,0 +1,174 @@
+# This manifest deploys a StarRocks cluster with all supported features.
+# If you want to understand the meaning of the fields in the following custom resource, please
+# refer to the explanations in other examples.
+
+apiVersion: starrocks.com/v1
+kind: StarRocksCluster
+metadata:
+  name: starrockscluster-sample
+spec:
+  starRocksFeSpec:
+    image: starrocks/fe-ubuntu:latest
+    replicas: 3
+    limits:
+      cpu: 8
+      memory: 16Gi
+    requests:
+      cpu: 8
+      memory: 16Gi
+    configMapInfo:
+      configMapName: starrockscluster-sample-fe-cm
+      resolveKey: fe.conf
+    storageVolumes:
+    - name: fe-storage-meta
+      storageClassName: "openebs-hostpath"  # you can remove this line if you want to use the default storage class
```

Comment:
this storage class is not universally available, we can provide examples of storage class on different platform, e.g. aliyun/aws/gke/azure, guide the end user to choice the right storage class.

Reply:
Good point, this can be done in a later PR.